cloud/wfm-rest-api/README.md

```diff
@@ -0,0 +1,108 @@
+# RainCatcher Api module
+
+Module used to expose express based api for WFM objects.
+
+### WFM speficic implementations
+
+Following api is being exposed:
+
+- `/workorders`
+- `/workflows`
+- `/results`
+
+This api is being added to new express router and it can be applied to any existing express based application
+as follows:
+
+```typescript
+// Create api
+const api = new WfmRestApi();
+
+// Mount api into path
+app.use('/api', api.createWFMRouter());
+```
+
+Api requires mongodb connection that needs to be setup as separate step
+
+```typescript
+api.setDb(db);
+```
+See demo application integration or [example application](./example) for more details.
+
+### Custom database integrations
+
+Custom database integrations are possible thanks to `CrudRepository` interface.
+See Api documentation for more details.
+
+## Rest API
+
+Module provides a way to dynamically create API for different business objects.
+Created api will use simplied implementations for typical create, read, delete and update operations.
+It's not recomended to use it outside WFM framework to store user related objects. Please use mongodb driver directly.
```

Comment:
1) What does "it" refer to in "to use it" - the rest api?
2) outside "the" WFM framework